microsite/data/plugins/dynatrace-managed.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
the content of this file looks like the same as `dynatrace.yaml`.
Is there any missing commit @johannes-b? 🤔

Reply:
@vinzscam you are absolutely right. I forgot to bring the updated `dynatrace.yaml`. Please take a look now.

Reply:
@vinzscam can you please help me to figure out why the workflow is failing at `yarn install`? Unfortunately, I can`t figure out why my changes break the workflow.

Reply:
let's seeeee probably a bad merge. I've rebased and force pushed the branch

Reply:
Thanks for your help and force pushing the PR. After doing a `prettier:check` the PR passed the checks. I would appreciate a final review.